spec.html

```diff
@@ -32580,18 +32580,19 @@ <h1>Date Objects</h1>
-        <p>A Number can exactly represent all integers from -9,007,199,254,740,992 to 9,007,199,254,740,992 (<emu-xref href="#sec-number.min_safe_integer"></emu-xref> and <emu-xref href="#sec-number.max_safe_integer"></emu-xref>). A time value supports a slightly smaller range of -8,640,000,000,000,000 to 8,640,000,000,000,000 milliseconds. This yields a supported time value range of exactly -100,000,000 days to 100,000,000 days relative to midnight at the beginning of 1 January 1970 UTC.</p>
-        <p>The exact moment of midnight at the beginning of 1 January 1970 UTC is represented by the time value *+0*<sub>𝔽</sub>.</p>
+        <p>An ECMAScript <dfn variants="epoch time values">epoch time value</dfn> is a Number, either a finite integral Number representing an instant in time to millisecond precision or *NaN* representing no specific instant. An epoch time value that is a multiple of <emu-eqn>24 × 60 × 60 × 1000 = 86,400,000</emu-eqn> (i.e., is 86,400,000 × _d_ for some integer _d_) represents the instant at the start of the UTC day that follows the epoch by _d_ whole UTC days (preceding the epoch for negative _d_). Every other finite epoch time value _t_ is defined relative to the greatest preceding epoch time value _s_ that is such a multiple, and represents the instant that occurs within the same UTC day as _s_ but follows it by (_t_ - _s_) milliseconds.</p>
+        <p>Epoch time values do not account for UTC leap seconds—there are no epoch time values representing instants within positive leap seconds, and there are epoch time values representing instants removed from the UTC timeline by negative leap seconds. However, the definition of epoch time values nonetheless yields piecewise alignment with UTC, with discontinuities only at leap second boundaries and zero difference outside of leap seconds.</p>
+        <p>A Number can exactly represent all integers from -9,007,199,254,740,992 to 9,007,199,254,740,992 (<emu-xref href="#sec-number.min_safe_integer"></emu-xref> and <emu-xref href="#sec-number.max_safe_integer"></emu-xref>). An epoch time value supports a slightly smaller range of -8,640,000,000,000,000 to 8,640,000,000,000,000 milliseconds. This yields a supported epoch time value range of exactly -100,000,000 days to 100,000,000 days relative to midnight at the beginning of 1 January 1970 UTC.</p>
+        <p>The exact moment of midnight at the beginning of 1 January 1970 UTC is represented by the epoch time value *+0*<sub>𝔽</sub>.</p>
+        <p>An ECMAScript <dfn variants="local time values">local time value</dfn> is an epoch time value adjusted by a time zone offset. Time zone offsets are integers in the exclusive range -86,400,000 to 86,400,000.</p>
```

Comment:
Two comments here:
1. We have been missing a good explanation of "local" for a while, and this is a good place to include it.
2. "[interval](https://tc39.es/ecma262/multipage/notational-conventions.html#interval)" is probably better than "exclusive range".

Feel free to tweak:
```suggestion
        <p>An ECMAScript <dfn variants="local time values">local time value</dfn> is an epoch time value adjusted by a time zone offset that is an integer number of milliseconds in the interval from -86,400,000 (exclusive) to 86,400,000 (exclusive). A local time value represents a local date and time of day, specifically the UTC date and time of day corresponding an identical epoch time value, ignoring range limits. For example, a local time value of *60000*<sub>𝔽</sub> represents time of day 00:01 on 1 January 1970, and could equivalently come from any of the following:</p>
        <ul>
          <li>local offset +00:00 = *+0*<sub>𝔽</sub> at epoch time 1970-01-01T00:01Z = *60000*<sub>𝔽</sub></li>
          <li>local offset +01:00 = *3600000*<sub>𝔽</sub> at epoch time 1969-12-31T23:01Z = 𝔽(-msPerHour + 60000) = *-3540000*<sub>𝔽</sub></li>
          <li>local offset -05:00 = 𝔽(-5 × msPerHour) = *-18000000*<sub>𝔽</sub> at epoch time 1970-01-01T05:01Z = 𝔽(5 × msPerHour + 60000) = *18060000*<sub>𝔽</sub></li>
        </ul>
```

Reply:
Added as 81aa87e2c8ea1419ba844a4383245836c5ad4ce1, can later be merged with 11579983a9f6c51d6c7b3adc22557dc4ff29381c.